features/steps/steps.py

```diff
@@ -221,3 +249,41 @@ def step_impl(context, related, relating, other_entity, condition):
                         errors.append(instance_structure_error(inst, rel.RelatingObject))
 
     handle_errors(context, errors)
+
+@then('The relative placement of that {entity} must be provided by an {other_entity} entity')
+def step_impl(context, entity, other_entity):
+    if getattr(context, 'applicable', True):
+        errors = []
+        for obj in context.instances:
+            if not obj.ObjectPlacement.is_a(other_entity):
+                errors.append(instance_placement_error(obj, other_entity, "", "", "", ""))
+        handle_errors(context, errors)
+
+
+@then(
+    'The {entity} entity must point to the {other_entity} of the container element established with {relationship} relationship')
```

Comment:
`other_entity` appears unused.

Reply:
I'm also thinking of whether it's necessary to repeat the `relationship` (=IfcRelAggregates) here and see what it would mean in a general case. Do you think it would also be possible to recover the IfcRelationship using the stack based approach? https://github.com/buildingSMART/ifc-gherkin-rules/issues/22 That might also help when there are multiple relationships of that type pointing to entity?

Reply:
Similar to my comment above, not every relationship means *containment*. I would even argue that `IfcRelAggregates` does not *contain*, but *aggregates*; there are other relationships for containment. Consider rewording.